cci_nepal/analysis/dummy_data/dummy_data_modeling/model_analysis.py

```diff
@@ -0,0 +1,295 @@
+import pandas as pd
+import numpy as np
+from numpy import absolute
+from numpy import mean
+from numpy import std
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import accuracy_score
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import cross_val_score
+from sklearn.model_selection import RepeatedKFold
+from sklearn.feature_selection import SequentialFeatureSelector
+from matplotlib import pyplot
+from operator import itemgetter
+
+import logging
+import cci_nepal
+from cci_nepal.getters.dummy_data import get_dummy_data as gdd
+
+
+def label_transformer(d):
+    if d < 1.5:
+        return "Unnecessary"
+    elif d < 2.5:
+        return "Desirable"
+    else:
+        return "Essential"
+
+
+def numeric_score_transformer(d):
+    if d < 1.5:
+        return 1
+    elif d < 2.5:
+        return 2
+    else:
+        return 3
+
+
+project_dir = cci_nepal.PROJECT_DIR
+logging.info(project_dir)
+
+
+df = gdd.read_csv_file(f"{project_dir}/inputs/data/Dummy_Data_Full_Survey_Features.csv")
+df.head()
+
+
+df.columns
+
+# Mapping categories to numbers #
```

Comment:
Probably not an issue but when we have the full data and their are the free text suggestions for NFRI items, would the below function map references of the words Essential, Desirable or Unnecessary to 3,2,1? Unlikely situation though.